### PR TITLE
Update install-vamt.md

### PR DESCRIPTION
**Pull request by Karl-WE**
missed to align the information in requirements section with merged PR #9009

### CODE REVIEW

File: windows/deployment/volume-activation/install-vamt.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ You install VAMT as part of the Windows Assessment and Deployment Kit (ADK) for
 ### Requirements
 
 - [Windows Server with Desktop Experience](https://docs.microsoft.com/windows-server/get-started/getting-started-with-server-with-desktop-experience), with internet access (for the main VAMT console) and all updates applied
-- [Windows 10, version 1903 ADK](https://go.microsoft.com/fwlink/?linkid=2086042)
+- the latest version of [Windows 10 ADK](https://docs.microsoft.com/windows-hardware/get-started/adk-install
```
Comment (illfated) [inline windows/deployment/volume-activation/install-vamt.md:36]: This sentence needs some form of rephrasing, it feels incomplete to me.
```suggestion
- Latest version of the [Windows 10 ADK](https://docs.microsoft.com/windows-hardware/get-started/adk-install)
```
@JohanFreelancer9 : May I ask you to suggest a correction or rephrasing to make it seem more natural? To me, it feels like "Windows 10 ADK" refers specifically to the Development Kit, hence my suggestion of the definite article "the" in front of "Windows 10 ADK".

Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/install-vamt.md:36]:
```suggestion
- Latest version of the [Windows 10 ADK](https://docs.microsoft.com/windows-hardware/get-started/adk-install)
```

Comment (JohanFreelancer9) [reply windows/deployment/volume-activation/install-vamt.md:36]: @illfated - I am good with your changes.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's changes.

**Comment by illfated** [comment]
@JohanFreelancer9 : Thank you for your valuable feedback.

@Karl-WE : Merging my suggestion should be enough.

**Comment by Karl-WE** [comment]
Aye. Thanks again. I wonder if I will ever achieve this level of written english :)

**Comment by illfated** [comment]
😄 Don't worry too much about it. There will always be someone available to proof read anything you write, if correct grammar, syntax, and semantics becomes important in your work.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
- **Regular source view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9179/files
- **Rich Diff view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9179/files?short_path=510612e

**Comment by illfated** [comment]
#sign-off

**Comment by Karl-WE** [comment]
Hi @greg-lindsay do you agree on the small change here? It would save time in the future, even if there will be only one ADK per year and I don't suspect a successor of ADK 2004 before 21H2. Thanks for your thoughts and merge.

**Comment by illfated** [comment]
Nice. Thank you both for the PR and for the merge. 👍🏻

**Comment by Karl-WE** [comment]
Thanks @greg-lindsay

**Pull request merged**